src/main/java/uk/gov/hmcts/reform/prl/services/gatekeeping/ListOnNoticeService.java

```diff
@@ -15,12 +15,11 @@ public String getReasonsSelected(Object listOnNoticeReasonsEnum,long caseId) {
         if (null != listOnNoticeReasonsEnum) {
             final String[] reasonsSelected = {""};
             ((List<String>) listOnNoticeReasonsEnum).stream().forEach(reason ->
-                reasonsSelected[0] = reasonsSelected[0].concat(ListOnNoticeReasonsEnum.getValue(reason) + "\n"));
-            log.info("***Reasons selected for list on Notice : {}", reasonsSelected[0]);
+                reasonsSelected[0] = reasonsSelected[0].concat(ListOnNoticeReasonsEnum.getDisplayedValue(reason) + "\n"));
             return reasonsSelected[0];
         } else {
             log.info("***No Reasons selected for list on Notice for the case id: {}", caseId);
-            return "";
+            return null;
         }
```

Comment:
do we need else case here . pls check and remove?

Reply:
Added logger .. so , else block added.. Let me know if I need to remove logger and else still.